### ISSUE TITLE
Build: Windows build metadata is incomplete or incorrect

### ISSUE THREAD

**Issue by scrubbbbs** [nomacs/nomacs#1209]
We should update the PE metadata, some of it is missing or has strange values. To the extent possible (I don't know about virustotal issues or others) the following files should all have matching metadata.

nomacs.exe
nomacsCore.dll
nomacs-setup-x64.exe
nomacs-setup-x64.msi

Note: metdata can be verified with sigcheck.exe tool from sysinternals

**Comment by novomesk** [comment]
Are you referring to VERSION INFO?

There is info about original developers who developed nomacs in the past:

Company: `TU Wien`
Copyright: `(C) 2011-2020, Markus Diem, Stefan Fiel, Florian Kleber`

What information should be there?

**Comment by scrubbbbs** [comment]
> What information should be there?

Something like:

Publisher: github.com/nomacs/nomacs
Company: The nomacs Project
Product: nomacs - Image Lounge
Description: free, open-source, cross-platform image viewer/editor | nomacs.org
Copyright: (C) 2011-2024 The nomacs Developers

**Comment by novomesk** [comment]
We can update the info, but I prefer to keep Markus Diem and Stefan Fiel explicitly mentioned as biggest part of the software is still their work.

GIMP for example still mentions people who started the project:
`Spencer Kimball, Peter Mattis and the GIMP Development Team`

**Comment by scrubbbbs** [comment]
I agree, we should give credit to the founders. Maybe we shouldn't take Florian's name off either, but you know more of the history than me so I'll accept your decision. I think it would be good if this matched the copyright on the about box and license as well.
